roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -19,77 +12,14 @@ auth supported = none
 fsid = {{ fsid }}
 {% endif %}
 max open files = {{ max_open_files }}
-osd pool default pg num = {{ pool_default_pg_num }}
-osd pool default pgp num = {{ pool_default_pgp_num }}
-osd pool default size = {{ pool_default_size }}
-osd pool default min size = {{ pool_default_min_size }}
-osd pool default crush rule = {{ pool_default_crush_rule }}
 {% if common_single_host_mode is defined %}
 osd crush chooseleaf type = 0
 {% endif %}
-{% if disable_in_memory_logs %}
-# Disable in-memory logs
-debug_lockdep = 0/0
-debug_context = 0/0
-debug_crush = 0/0
-debug_buffer = 0/0
-debug_timer = 0/0
-debug_filer = 0/0
-debug_objecter = 0/0
-debug_rados = 0/0
-debug_rbd = 0/0
-debug_journaler = 0/0
-debug_objectcatcher = 0/0
-debug_client = 0/0
-debug_osd = 0/0
-debug_optracker = 0/0
-debug_objclass = 0/0
-debug_filestore = 0/0
-debug_journal = 0/0
-debug_ms = 0/0
-debug_monc = 0/0
-debug_tp = 0/0
-debug_auth = 0/0
-debug_finisher = 0/0
-debug_heartbeatmap = 0/0
-debug_perfcounter = 0/0
-debug_asok = 0/0
-debug_throttle = 0/0
-debug_mon = 0/0
-debug_paxos = 0/0
-debug_rgw = 0/0
-{% endif %}
-{% if enable_debug_global %}
-debug ms = {{ debug_global_level }}
-{% endif %}
-
 [client]
-rbd cache = {{ rbd_cache }}
-rbd cache writethrough until flush = true
```

Comment:
There is a default for `rbd_cache_writethrough_until_flush` despite it not actually getting used.